openshift/release/resolve.sh

```diff
@@ -21,7 +21,7 @@ function resolve_file() {
   # 1. Rewrite image references
   # 2. Update config map entry
   # 3. Replace serving.knative.dev/release label.
-  sed -e "s+serving.knative.dev/release: devel+serving.knative.dev/release: \"v0.26.0\"+" \
-      -e "s+app.kubernetes.io/version: devel+app.kubernetes.io/version: \"v0.26.0\"+" \
+  sed -e "s+serving.knative.dev/release: devel+serving.knative.dev/release: \"knative-v1.0.1\"+" \
+      -e "s+app.kubernetes.io/version: devel+app.kubernetes.io/version: \"knative-v1.0.1\"+" \
```

Comment:
@markusthoemmes Where can I compare the version value here with the ones used in Serverless Operator?